README.md

```diff
@@ -148,7 +153,7 @@ Then recreate the completion script:
 
 ```shell script
 java -cp "target/quarkus-app/app/*:target/quarkus-app/lib/main/*:target/quarkus-app/quarkus-run.jar" \
-  picocli.AutoComplete -n kcctl --force dev.morling.kccli.command.KcCtlCommand
+  picocli.AutoComplete -n kcctl --force org.moditect.kcctl.command.KcCtlCommand
```

Comment:
Ah yes, thx. Btw. I goofed up when doing that package rename, this should be `org.kcctl.*` actually. [ModiTect](https://github.com/moditect/) is another open-source effort I'm involved with, but it's unrelated to kcctl.